bucket/cuda.json

```diff
@@ -35,11 +35,11 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://developer.download.nvidia.com/compute/cuda/$majorVersion.$minorVersion/Prod/local_installers/cuda_$version_win10.exe#/dl.7z"
+                "url": "http://developer.download.nvidia.com/compute/cuda/$matchHead/local_installers/cuda_$version_win10.exe#/dl.7z"
             }
         },
         "hash": {
-            "url": "http://developer.download.nvidia.com/compute/cuda/$majorVersion.$minorVersion/Prod/docs/sidebar/md5sum.txt"
+            "url": "https://developer.download.nvidia.com/compute/cuda/$matchHead/docs/sidebar/md5sum.txt"
```

Comment:
The other two changes are https > http. Is this change from http to https correct?

Reply:
In page [CUDA Toolkit 11.0 Update 1 Downloads](https://developer.nvidia.com/cuda-downloads?target_os=Windows&target_arch=x86_64&target_version=10&target_type=exelocal), the link is http

Reply:
So, shouldn’t the autoupdate link be left as http?